def print_error(bot, update, error):
    print('Update {} caused error {}'.format(update, error))
